src/components/ThoughtAnnotation.tsx

```diff
@@ -49,7 +49,7 @@ const getSubThoughtTextMarkup = (state: State, isEditing: boolean, subthought: {
   const { editingValue } = state
   return {
     __html: isEditing
-      ? editingValue && subthought.text !== editingValue ? editingValue : subthought.text
+      ? editingValue && subthought.text === editingValue ? editingValue : subthought.text
```

Comment:
`editingValue && subthought.text === editingValue ? editingValue : subthought.text ` is equivalent to `subthought.text`.

This code is intended to use the live value in the `ThoughtAnnotation` when it is being edited. What's purpose of this change? It seems like removing `editingValue` would eliminate existing functionality.

Reply:
Hmmm
Sometimes we have such render after `splitSentences`. In this case `editingValue` is `1.2.` and `subthought.text` is `1.` Maybe we should just set correct `editingValue` in the last step of `splitSentences` reducer.
![Screenshot from 2020-10-07 12-42-37](https://user-images.githubusercontent.com/32064251/95291957-c4857b00-089a-11eb-9e71-a562da5a8e52.png)

Reply:
> Maybe we should just set correct `editingValue` in the last step of `splitSentences` reducer.

Yes, try that and see if it works.

`editingValue` should be [set on focus](https://github.com/cybersemics/em/blob/dev/src/components/Editable.tsx#L511) but maybe that's too late.